scripts/setup_rdma_env.sh

```diff
@@ -0,0 +1,137 @@
+#/usr/bin/env bash
+
+if [[ ! -d rdma ]]; then
+    mkdir rdma
+fi
+cd rdma
+
+install_ice() {
+    echo "Installing ice driver..."
+    wget https://downloadmirror.intel.com/832286/ice-1.15.4.tar.gz -O ice.tar.gz
+    tar -xf ice.tar.gz
+    cd ice-*/src
+    sudo make install
+    cd ../..
+}
```

Comment:
Ok this will break the whole stack - and user will have it hard to debug - as we use patched versions of Intel ICE drivers, and without those, the MTL and MCM will not work as intended/will not work at all:

https://github.com/OpenVisualCloud/Media-Transport-Library/tree/main/patches/ice_drv

Reply:
You are right. I will assume that the ice driver is installed and remove its installation from the script